PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -1,7 +1,7 @@
 javaVersion=1.8
 org.gradle.jvmargs='-Duser.language=en'
 sources=false
-intellij_version=IC-LATEST-EAP-SNAPSHOT
+intellij_version=IC-2019.2.4
```

Comment:
quick question, why we keep the `intellij_version` as the latest EAP version before?

Reply:
Because we keep two version build before.